k8s/overlays/azure/test/keda/cse-import-ec-d2cc-scale.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: cse-import-ec-d2cc-data-bridge-scale
+  namespace: gridcapa-t
+spec:
+  scaleTargetRef:
+    name: cse-import-ec-d2cc-data-bridge
+  minReplicaCount: 0
+  maxReplicaCount: 1
+  triggers:
+    - type: cron
+      metadata:
+        timezone: "UTC"
+        start: "0 7 * * 1-5"
+        end: "0 17 * * 1-5"
+        desiredReplicas: "1"
+
+---
+
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: cse-import-ec-d2cc-export-scale
+  namespace: gridcapa-t
+spec:
+  scaleTargetRef:
+    name: cse-import-ec-d2cc-export
```

Comment:
the same here

Reply:
As discussed, the scaleTargetRef points to the existing deployment which is currently named **cse-import-ec-d2cc-export** all over gridcapa-deployment. Perhaps we could rename this everywhere in another ticket